### DISCUSSION TITLE
error: unable to read input file: *.pdb

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#1459] [Model evaluation and manipulation]
On 2016-05-04 11:17:50, user 2107086a wrote:

Hi,

I'm trying to run multiple pdbs in CRYSOL but I keep on getting messages 'error: unable to read input file: *.pdb'. I also have the CRYSOL3 option in my ATSAS package but I'm not sure if it runs teh same as CRYSOL..Any ideas as to why this happens?

**Comment by franke-biosaxs** [comment]
On 2016-05-04 13:06:32, user ckerr wrote:

Which OS are you using, and what exact command line are you typing?

**Comment by franke-biosaxs** [comment]
On 2016-05-04 17:01:13, user 2107086a wrote:

OS 10.10.5 and the command I'm using is

$ "/Applications/ATSAS/bin/crysol *.pdb bd1971.dat -sm 0.3"

**Comment by franke-biosaxs** [comment]
On 2016-05-09 08:17:23, user ckerr wrote:


2107086a wrote
> OS 10.10.5 and the command I'm using is
> 
> $ "/Applications/ATSAS/bin/crysol *.pdb bd1971.dat -sm 0.3"

I'm assuming you mean ```
$ /Applications/ATSAS/bin/crysol *.pdb bd1971.dat -sm 0.3
```
(without the quotation marks)
```
*.pdb
```
means "all files with names ending in '.pdb' in the current directory". This so-called 'glob expansion' is done by the command shell and not by crysol itself - the command line that crysol sees will be something like ```
crysol A.pdb B.pdb C.pdb bd1971.dat -sm 0.3
```
. If there are no files ending in .pdb, or if you put the ```
*.pdb
```
in quote marks ( ```
"*.pdb"
```
), the glob expansion will not be done and crysol will try to open a file called '*.pdb', but this file does not exist.

In short, you need to run the command from the folder where the pdb files are stored.
